application/blocks/heading-block.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace CNET\Bridge\Blocks;
+
+class Heading_Block extends Block_Parser implements Block_Interface {
+	const BLOCK_TYPE = 'core/heading';
+
+	public function __construct() {
+		add_action('init',function () {
+			$this->init();
+		});
+	}
+
+	function init() {
+		add_filter('bridge_block_filter', [$this, 'block_parser']);
+	}
+
+	function block_parser( $content ) {
+		$this->blocks = $content;
+
+		$filteredBlocks = $this->filter_by_block_name(self::BLOCK_TYPE);
+
+		return $this->to_html($filteredBlocks);
+	}
+
+	function to_html( $filteredBlocks ) {
+		foreach ($filteredBlocks as $key => $block) {
+			$this->blocks[$key] = [
+				'blockName' => 'parsed',
+				'block_html' => $this->get_attributes($block)
+			];
+		}
+
+		return $this->blocks;
+	}
+
+	private function get_attributes($block) {
```

Comment:
shouldn't we try and move this to the parent class? We've used it in 2 blocks so far

Reply:
@ocorralesRV added function to interface